session-signatures/getLitActionSessionSigs/README.md

```diff
@@ -0,0 +1,42 @@
+# `getLitActionSessionSigs` Code Example
+
+This code demonstrates how to use the `getLitActionSessionSigs()` method from the Lit SDK.
+
+## Understanding the Implementation
+1. Using an imported Ethereum private key, connect the wallet to the Lit RPC endpoint `Chronicle Yellowstone`
+2. Connect to the Lit network using the `LitNodeClient` on the `datil-dev` network, specifying the local storage to hold the generated wallet and session keys
+3. Connect the `LitContracts` client to the Lit network
+4. Mint a PKP using the `pkpNftContractUtils.write.mint()` method from `LitContracts`. You may receive a gas error if your `ethersSigner` has insufficient tokens. This can be fixed by acquiring more tokens for the `Chronicle Yellowstone` blockchain
```

Comment:
```suggestion
4. Mint a PKP using the `pkpNftContractUtils.write.mint()` method from `LitContracts`.
    - You may receive a gas error if your `ethersSigner` has insufficient tokens. This can be fixed by acquiring more tokens for the `Chronicle Yellowstone` blockchain
```